### PR TITLE
new 3.0 sets

### PR DESCRIPTION
**Pull request by imring**

### CODE REVIEW

File: pkg/shortcut/artifacts.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ var SetNameToKey = map[string]keys.Set{
 	"cw":                     keys.CrimsonWitchOfFlames,
 	"crimson":                keys.CrimsonWitchOfFlames,
 	"witch":                  keys.CrimsonWitchOfFlames,
+	"deepwoodmemories":       keys.DeepwoodMemories,
```
Comment (k0l11) [inline pkg/shortcut/artifacts.go:23]: pls add "deepwood" as alias

File: pkg/shortcut/artifacts.go [hunk 2]
```diff
@@ -33,6 +34,7 @@ var SetNameToKey = map[string]keys.Set{
 	"gladiatorsfinale":       keys.GladiatorsFinale,
 	"glad":                   keys.GladiatorsFinale,
 	"gladiators":             keys.GladiatorsFinale,
+	"gildeddreams":           keys.GildedDreams,
```
Comment (k0l11) [inline pkg/shortcut/artifacts.go:37]: pls add "gilded" and "gd" as alias

File: pkg/shortcut/artifacts.go [hunk 3]
```diff
@@ -35,6 +36,8 @@ var SetNameToKey = map[string]keys.Set{
 	"glad":                   keys.GladiatorsFinale,
 	"gladiators":             keys.GladiatorsFinale,
 	"gildeddreams":           keys.GildedDreams,
+	"glided":                 keys.GildedDreams,
```
Comment (k0l11) [inline pkg/shortcut/artifacts.go:39]: should be "gilded"

File: internal/artifacts/gildeddreams/gildeddreams.go [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+		add := func(args ...interface{}) bool {
+			atk := args[1].(*combat.AttackEvent)
+			if c.Player.Active() != char.Index {
+				return false
+			}
+			if atk.Info.ActorIndex != char.Index {
+				return false
+			}
+
+			s.emCount = 0
```
Comment (srliao) [inline internal/artifacts/gildeddreams/gildeddreams.go:60]: this section here should just be cached outside of the function. no real reason to recompute every time the reaction gets triggered

Comment (srliao) [reply internal/artifacts/gildeddreams/gildeddreams.go:60]: keep it as part of the fields for `s` though i think

**Review by srliao — APPROVED** [review]

**Pull request merged**